### DISCUSSION TITLE
Grads output error

### DISCUSSION THREAD

**Discussion by meenakshisreejith** [NOAA-EMC/WW3#567] [Q&A]
Hello all
I am trying to generate grads output after successful run for five years. I have changed the start date in gx_outf.inp and my time interval is three hours. but I am getting error as: 

Open Error:  Missing or invalid dimension size.
  --> The invalid description file record is: 
--> TDEF     ****  LINEAR     00:00Z01DEC1979   3HR
  The data file was not opened. 

cdo    import_binary (Abort): Open failed!

[gx_outf.txt](https://github.com/NOAA-EMC/WW3/files/7800442/gx_outf.txt)

I have attached my gx_outf.inp file as a text file. Please help me in this regard.

**Comment by JessicaMeixner-NOAA** [comment]
@aliabdolali have you used the grads scripts before? Can you help @meenakshisreejith with this?

**Reply by JessicaMeixner-NOAA** [reply]
@meenakshisreejith I looked at your input file and it looks okay to me however, I don't use grads, so I'm not a lot of help with this issue.  Two things I'd recommend: one try only doing HS and seeing if that works, and two maybe look at which cdo library you are using?
